src/styles/index.ts

```diff
@@ -4284,6 +4284,11 @@ const styles = (theme: ThemeColors) =>
             overflow: 'hidden',
         },
 
+        walletCardNumber: {
+            color: theme.text,
+            fontSize: variables.fontSizeNormal,
+        },
```

Comment:
This style was missing before, but I am fine with using default text color and font size.
Just commenting here in case someone reports any style regression pointing to this PR.